src/schema/schema.js

```diff
@@ -0,0 +1,60 @@
+const { buildSchema } = require('graphql');
+
+const schema = buildSchema(`
+  type User {
+    id: ID!
+    avatar_url: String!
+    fetchOrgs: [Org]
```

Comment:
@michaelmurray6298 I would recommend this to simply be `orgs` rather than `fetchOrgs`